<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSortOptionDlg</class>
 <widget class="QDialog" name="KSortOptionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sort options</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QCheckBox" name="m_useDefault">
     <property name="text">
      <string>Use default</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TransactionSortOption" name="m_sortOption" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TransactionSortOption</class>
   <extends>QWidget</extends>
   <header>transactionsortoption.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_useDefault</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_sortOption</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KSortOptionDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KSortOptionDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
